client/my-sites/email/email-management/titan-control-panel-login-card/index.jsx

```diff
@@ -0,0 +1,94 @@
+/**
+ * External dependencies
+ */
+import { localize } from 'i18n-calypso';
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import wpcom from 'lib/wp';
+import { errorNotice } from 'state/notices/actions';
+import { Button, CompactCard } from '@automattic/components';
+import SectionHeader from 'components/section-header';
+
+class TitanControlPanelLoginCard extends React.Component {
+	state = {
+		isFetchingAutoLoginLink: false,
+	};
+
+	fetchTitanAutoLoginURL = ( orderId ) => {
+		return new Promise( ( resolve ) => {
+			wpcom.undocumented().getTitanControlPanelAutoLoginURL( orderId, ( serverError, result ) => {
+				resolve( {
+					error: serverError?.message,
+					loginURL: serverError ? null : result.auto_login_url,
+				} );
+			} );
+		} );
+	};
+
+	onLogInClick = () => {
+		if ( this.state.isFetchingAutoLoginLink ) {
+			return;
+		}
+
+		const { translate } = this.props;
+		this.setState( { isFetchingAutoLoginLink: true } );
+
+		// TODO: use actual orderID
+		this.fetchTitanAutoLoginURL( 12345 ).then( ( { error, loginURL } ) => {
+			this.setState( { isFetchingAutoLoginLink: false } );
+			if ( error ) {
+				this.props.errorNotice(
+					error ?? translate( 'An unknown error occurred. Please try again later.' )
+				);
+			} else {
+				window.location.href = loginURL;
```

Comment:
This looks like provider's URL will take over the window. Could we look at sending them to a new tab/window? Is that something that makes sense?

Reply:
We could check our other third-party integrations (such as Lookea) to see how that is handled. @fditrapani may also have suggestions.

Reply:
Yeah well... there's the possibility of it being blocked as popup. Otherwise we'd have to load the link ahead of time, which I don't think would be a good idea.

Reply:
In the past, we have struggled with opening too many tabs which can get overwhelming for people. For that reason and what @eltongo mentioned, my vote is to keep it in the same window/tab. People can always use the back button and open it in a new tab if they desire.